gen/dcos-config.yaml

```diff
@@ -4,6 +4,7 @@ package:
       {
         "environment": {
           "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip",
+          "PLATFORM": "{{ platform }}"
```

Comment:
If I delete this, will it affect https://github.com/mesosphere/dcos/blob/d39931d6d30815874c32b100b5824185d5dd3a4e/packages/dcos-integration-test/extra/test_env.export#L13 ?